live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/hx_insight/storage/connector/HttpFileUploader.java

```diff
@@ -95,18 +96,34 @@ public void configure()
             backoff = @Backoff(delayExpression = "#{@integrationProperties.hylandExperience.storage.upload.retry.initialDelay}",
                     multiplierExpression = "#{@integrationProperties.hylandExperience.storage.upload.retry.delayMultiplier}"))
     @Override
+    @SuppressWarnings({"PMD.CloseResource", "PMD.PreserveStackTrace"})
     public void upload(FileUploadRequest fileUploadRequest)
     {
-        Map<String, Object> headers = Map.of(STORAGE_LOCATION_HEADER, wrapRawToken(fileUploadRequest.storageLocation()),
-                Exchange.CONTENT_TYPE, fileUploadRequest.contentType());
-
-        try (InputStream fileData = fileUploadRequest.file().data())
+        InputStream fileData = fileUploadRequest.file().data();
+        try
```

Comment:
Is there any particular reason you stopped using try-with-resources here? It might be better if the `InputStream` is automatically closed.
Ok - I see it is reset in `catch` clause. Still, maybe it'd be worth closing the `InputStream` in `finally` block?